builder/amazon/chroot/builder.go

```diff
@@ -287,7 +287,7 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, []string, error) {
 	errs = packersdk.MultiErrorAppend(errs, b.config.RootVolumeTag.CopyOn(&b.config.RootVolumeTags)...)
 	errs = packersdk.MultiErrorAppend(errs, b.config.SourceAmiFilter.Prepare()...)
 
-	errs = packersdk.MultiErrorAppend(errs, b.config.AccessConfig.Prepare(&b.config.ctx)...)
```

Comment:
Huh... why were we passing context in here in the first place?

Reply:
It wasn't being used to I just removed it. 
I believe it was there because once in the past the method did some sort of interpolation [[here](https://github.com/hashicorp/packer/commit/96cc8a4aee493b9d080a586740286b6f9ffd71f3)].